client/src/components/screens/PatientsPrescriptionsSearch/table/prescriptionColumns.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import intl from 'react-intl-universal';
+import { Link } from 'react-router-dom';
+import { ISyntheticSqon } from '@ferlab/ui/core/data/sqon/types';
+import { addToSqons } from '@ferlab/ui/core/data/sqon/utils';
+import { Badge, Button } from 'antd';
+
+import './tableColumn.scss';
+
+export type TStudyColumn = {
+  title: string | JSX.Element;
+  dataIndex?: string | string[];
+  name?: string | string[];
+  width?: number;
+  summary?: boolean;
+  render?: (arg1: any, arg2?: any) => JSX.Element | string;
+  children?: TStudyColumn[];
+  key: string;
+};
+
+const statusColors: Record<string, Record<string, string>> = {
+  'active': {
+    color: '#1D8BC6',
```

Comment:
are these colours global? If so, maybe we could extract them in a more general place.